src/plugins/safeSnap/index.ts

```diff
@@ -126,6 +126,40 @@ export default class Plugin {
     yield;
   }
 
+  async *deleteDisputedProposal(
+    web3: any,
+    moduleAddress: string,
+    transactionHash: any
+  ) {
+    const tx = await sendTransaction(
+      web3,
+      moduleAddress,
+      UMA_MODULE_ABI,
+      'deleteDisputedProposal',
+      [transactionHash]
+    );
+    yield;
```

Comment:
In the above function, a string description is returned from the first yield. Should we do that here?

Reply:
I'm not actually sure what the string yielded in the earlier function is doing. It might be used somewhere else within Snapshot's system but I don't see it in the SafeSnap-specific codebase.